src/heat-map/heat-map.component.ts

```diff
@@ -161,16 +162,18 @@ export class HeatMapComponent extends BaseChartComponent {
   }
 
   getXScale(): any {
+    const innerPadding = typeof this.innerPadding === 'number' ? this.innerPadding : this.innerPadding[0];
     return d3.scaleBand()
       .rangeRound([0, this.dims.width])
-      .paddingInner(0.1)
+      .paddingInner(innerPadding / this.dims.width)
       .domain(this.xDomain);
   }
 
   getYScale(): any {
+    const innerPadding = typeof this.innerPadding === 'number' ? this.innerPadding : this.innerPadding[1];
     return d3.scaleBand()
       .rangeRound([this.dims.height, 0])
-      .paddingInner(0.1)
+      .paddingInner(innerPadding / this.dims.height)
```

Comment:
Similar to the above, this needs to be

`innerPadding * this.yDomain.length / this.dims.height`